<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlanEditorClass</class>
 <widget class="ITaskEditor" name="PlanEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlanEditor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="title">
      <string>计划信息</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>名称：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_planName">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>plan_detect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,3">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>模式：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_planMode">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>水面侦察</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,3">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>总里程：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_planDistance">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>13000km</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,3">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>总耗时：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_planTime">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>87天12小时</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>最小深度：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_planMinDepth">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>0m</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>最大深度：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_planMaxDepth">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>300m</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0,3">
        <item>
         <widget class="QLabel" name="label_21">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>能源总消耗:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_planEnergy">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>9800kw/h</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,2">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>11</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>任务数量：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_taskCount">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ITaskEditor</class>
   <extends>QWidget</extends>
   <header>itaskeditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
